src/index.js

```diff
@@ -1,11 +1,12 @@
 import routerStateReducer from './routerStateReducer';
-import reduxRouteComponent from './reduxRouteComponent';
+import reduxRouteComponent, {routerMiddleware} from './reduxRouteComponent';
```

Comment:
Why not `import {reduxRouteComponent, routerMiddleware} from './reduxRouteComponent';`?